#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_cola-desktop.dpatch by Iulian Udrea <iulian@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Removed Encoding line because it's deprecated and modified the
## following lines: Name, Comment, Exec and Categories.

@DPATCH@
diff -urNad git-cola-1.2.0~/share/applications/cola.desktop git-cola-1.2.0/share/applications/cola.desktop
--- git-cola-1.2.0~/share/applications/cola.desktop	2008-09-22 02:32:21.000000000 +0100
+++ git-cola-1.2.0/share/applications/cola.desktop	2008-09-22 07:44:42.000000000 +0100
@@ -1,11 +1,10 @@
 [Desktop Entry]
-Encoding=UTF-8
-Name=cola
-Comment=cola: a highly caffeinated git gui
-Exec=git cola
+Name=Cola
+Comment=Highly caffeinated git gui
+Exec=git-cola
 Icon=/usr/share/cola/icons/git.png
 StartupNotify=true
 Terminal=false
 Type=Application
-Categories=Development
+Categories=Development;
 X-KDE-SubstituteUID=false
